app/assets/stylesheets/bookings/_index.scss

```diff
@@ -1,27 +1,13 @@
 .booking {
-  body {
-    display: flex;
```

Comment:
can i assume that this is being replaced by .flex-wrap-center class?

Reply:
Hmmm.. not replacing anything, just that it wasn't used hence I deleted :)